en/about/overview.md

```diff
@@ -63,5 +64,9 @@ In most cases where these fields are used the sub-protocol also ensures guarante
 
 ## Integrity Checks / Checksum
 
-MAVLink implements two integrity checks: The first check is on the integrity of the packet during transmission using the X.25 checksum ([CRC-16-CCITT](https://en.wikipedia.org/wiki/Cyclic_redundancy_check)). This however only ensures that the data has not been altered on the link - it does not ensure consistency with the data definition. The second integrity check is on the [data description](https://en.wikipedia.org/wiki/Data_definition_language) to ensure that two messages with the same ID are indeed containing the same information. To achieve this the data definition itself is run through CRC-16-CCITT and the resulting value is used to seed the packet CRC. Most reference implementations store this constant in an array named **CRC\_EXTRA**.
+MAVLink implements two integrity checks: The first check is on the integrity of the packet during transmission using the CRC-16/MCRF4XX checksum.
+This however only ensures that the data has not been altered on the link - it does not ensure consistency with the data definition.
+The second integrity check is on the [data description](https://en.wikipedia.org/wiki/Data_definition_language) to ensure that two messages with the same ID are indeed containing the same information.
+To achieve this the data definition itself is run through CRC-16-CCITT and the resulting value is used to seed the packet CRC.
```

Comment:
Are we ordering the description "badly". Ie I read this as we do CRC on the data definition first, and then we use the result as seed/part of the check for the transmission. We could perhaps word as first run the check on data definition, then generate further CRC using it to check for transmission.

Or to put it another way, should we describe this using the order of operations?